src/modules/setup/setupESLint.ts

```diff
@@ -21,42 +54,62 @@ const addToEslintrc = {
       jest: true,
     },
   },
-  node: {
-    extends: 'vtex',
-    env: {
-      node: true,
-      es6: true,
-      jest: true,
-    },
-  },
 }
 
-const yarnAddESLint = (relativePath: string) => {
-  log.info(`Adding lint configs in ${relativePath}`)
-  const lintDeps = R.join(' ', R.values(R.mapObjIndexed((version, name) => `${name}@${version}`, addToPackageJson)))
-  execSync(`${yarnPath} add ${lintDeps} --dev`, {
+const yarnAddESLint = () => {
+  log.info('Adding lint configs in app root')
+  const lintPackages = R.join(' ', R.values(R.mapObjIndexed((version, name) => `${name}@${version}`, lintDeps)))
+  execSync(`${yarnPath} add ${lintPackages} --dev`, {
     stdio: 'inherit',
-    cwd: resolvePath(getAppRoot(), `${relativePath}`),
+    cwd: resolvePath(getAppRoot()),
   })
 }
 
-const createESLintSetup = async (lintDeps: string[], builder: string) => {
+const createESLintSetup = (appName: string, lintPackages: string[]) => {
   try {
-    const devDependencies = R.prop('devDependencies', packageJsonEditor.read(builder)) || {}
-    if (R.difference(lintDeps, R.intersection(lintDeps, R.keys(devDependencies))).length !== 0) {
-      yarnAddESLint(builder)
+    const originalRootPackageJson = packageJsonEditor.read('.')
```

Comment:
I think something like this will be necessary here:
```typescript
  let originalRootPackageJson = {}
  try {
    originalRootPackageJson = packageJsonEditor.read('.')
  } catch (e) {
    if (e.code === 'ENOENT') {
      log.warn(`No package.json found in ${packageJsonEditor.path('.')}. Creating one.`)
    } else {
      log.error(e)
      return
    }
  }
```

Reply:
If a `package.json` doesn't exist in root an error is being thrown right away

Reply:
done! :-)